sources/sections-a1-en/18-16th-cgpm.adoc

```diff
@@ -43,11 +44,14 @@ The 16^th^ Conférence Générale des Poids et Mesures,
 * that the proliferation of special names represents a danger for the Système International d'Unités and must be avoided in every possible way, but that this rule can be broken when it is a matter of safeguarding human health,
 
 *adopts* the special name *_sievert_*, symbol stem:["Sv"], for the SI unit of dose equivalent in the field of radioprotection. The sievert is equal to the joule per kilogram.
+____
 
-NOTE: The CIPM, in 1984, decided to accompany this Resolution with an explanation (Recommendation 1, see p. 176).
+NOTE: The CIPM, in 1984, decided to accompany this Resolution with an explanation <<cipm1984r1,(Recommendation 1)>>.
```

Comment:
As above. Should be `(<<cipm1984r1,Recommendation 1>>)`

Reply:
Done.